tools/tertiary-analysis/scanpy/scanpy-filter-cells.xml

```diff
@@ -35,26 +42,27 @@ PYTHONIOENCODING=utf-8 scanpy-filter-cells
     <param name="gene_name" type="text" optional="true" label="Name of the column in `anndata.var` that contains gene name"
            help="Used for flagging mitochondria genes (starting with 'MT-'). Leave empty if gene table already has a boolean column called 'mito' that flags MT genes"/>
 
-    <repeat name="parameters" title="Parameters used to filter cells" min="1">
+    <repeat name="parameters" title="Parameters to select cells to keep" min="1">
       <param name="name" type="text" value="n_genes" label="Name of parameter to filter on" help="for example n_genes or n_counts">
         <option value="n_genes">n_genes</option>
         <option value="n_counts">n_counts</option>
-        <option value="pct_counts_mito">pct_counts_mito (only usable if MT genes are flagged)</option>
+        <option value="pct_counts_mito">pct_counts_mito (only usable if MT genes are flagged or has been pre-calculated)</option>
       </param>
-      <param name="min" type="float" value="0" min="0" label="Min value"/>
-      <param name="max" type="float" value="1e9" label="Max value"/>
+      <param name="min" type="float" value="0" min="0" label="Min value" help="Cells with value below min will be discarded."/>
```

Comment:
Should we put a note about use of percentages here? e.g. "Specify percentages as 1-100 rather than 0-1"?